CMake/lus-cvars.cmake

```diff
@@ -0,0 +1,52 @@
+#SoH defines
+set(CVAR_PREFIX_RANDOMIZER_ENHANCEMENT "gRandoEnhancements")
+set(CVAR_PREFIX_RANDOMIZER_SETTING "gRandoSettings")
+set(CVAR_PREFIX_COSMETIC "gCosmetics")
+set(CVAR_PREFIX_AUDIO "gAudioEditor")
+set(CVAR_PREFIX_CHEAT "gCheats")
+set(CVAR_PREFIX_ENHANCEMENT "gEnhancements")
+set(CVAR_PREFIX_SETTING "gSettings")
+set(CVAR_PREFIX_WINDOW "gOpenWindows")
+set(CVAR_PREFIX_TRACKER "gTrackers")
+set(CVAR_PREFIX_DEVELOPER_TOOLS "gDeveloperTools")
+set(CVAR_PREFIX_GENERAL "gGeneral")
+set(CVAR_PREFIX_REMOTE "gRemote")
+add_compile_definitions(
+	CVAR_PREFIX_RANDOMIZER_ENHANCEMENT="${CVAR_PREFIX_RANDOMIZER_ENHANCEMENT}"
+	CVAR_PREFIX_RANDOMIZER_SETTING="${CVAR_PREFIX_RANDOMIZER_SETTING}"
+	CVAR_PREFIX_COSMETIC="${CVAR_PREFIX_COSMETIC}"
+	CVAR_PREFIX_AUDIO="${CVAR_PREFIX_AUDIO}"
+	CVAR_PREFIX_CHEAT="${CVAR_PREFIX_CHEAT}"
+	CVAR_PREFIX_ENHANCEMENT="${CVAR_PREFIX_ENHANCEMENT}"
+	CVAR_PREFIX_SETTING="${CVAR_PREFIX_SETTING}"
+	CVAR_PREFIX_WINDOW="${CVAR_PREFIX_WINDOW}"
+	CVAR_PREFIX_TRACKER="${CVAR_PREFIX_TRACKER}"
+	CVAR_PREFIX_DEVELOPER_TOOLS="${CVAR_PREFIX_DEVELOPER_TOOLS}"
+	CVAR_PREFIX_GENERAL="${CVAR_PREFIX_GENERAL}"
+	CVAR_PREFIX_REMOTE="${CVAR_PREFIX_REMOTE}"
+)
```

Comment:
didn't notice this before apparently, but this file is called `lus-cvars.cmake` and it includes soh cvars, maybe move these soh ones to a new `soh-cvars.cmake`?

Reply:
I can do that.

Reply:
Done.